lib/utils/typeGuards.mjs

```diff
@@ -49,6 +49,14 @@ export function isDocument(node) {
 	return node.type === 'document';
 }
 
+/**
+ * @param {import('postcss-value-parser').Node} node
+ * @returns {node is import('postcss-value-parser').DivNode}
+ */
+export function isValueDiv(node) {
```

Comment:
minor
suggestion

Maybe update 

https://github.com/stylelint/stylelint/blob/68cb920c45641702d76670afb111960cbd743296/lib/rules/font-weight-notation/index.mjs#L188-L190

Reply:
Out of scope. Feel free to do it in another PR.